.settings.test.toml

```diff
@@ -45,4 +45,8 @@ basket_url = "http://localhost:4321"
 [playground]
 github_token = "foobar"
 crypt_key = "IXAe2h1MekK4LKysmMvxomja69PT6c20A3nmcDHQ2eQ="
-flag_repo = "flags"
\ No newline at end of file
+flag_repo = "flags"
+
+[ai]
```

Comment:
We should also add this section to the other `.settings.*.toml` files.

Reply:
Once they are not optional.

Reply:
Fixed in 54ed74c